### ISSUE TITLE
OSC release false positive

### ISSUE THREAD

**Issue by smarlowucf** [openSUSE/osc#860]
When releasing a multibuild package osc provides a false positive result when the multibuild target does not exist. See the example where the package and project are valid but the multibuild flavor is fake:

```
> osc token --trigger {token} --operation release {project} {valid package}:FAKE
Trigger token
<status code="ok">
  <summary>Ok</summary>
</status>
```

It seems like this should return an error result.

**Comment by JanZerebecki** [comment]
I suspect the response from the open build service server is positive because it only queues this action, so the failure will only be apparent in the future. To verify use of the debug flags to osc to see the API calls. AFAIK OBS has no API to get the real result once it is available nor one to wait for the result. Please file a feature request for this in https://github.com/openSUSE/open-build-service/issues and link the issue. After that is implemented osc could use it.

**Comment by smarlowucf** [comment]
No, it only provides a false positive in the event that you attempt to trigger an invalid multibuild profile. It will return a failed status if you trigger an invalid package.

This fails as expected:

```
osc token --trigger {token} --operation release {project} {invalid package}
```

This does not fail as expected:

```
osc token --trigger {token} --operation release {project} {valid package}:INVALID
```
